README.md

```diff
@@ -150,6 +150,15 @@ To define the options for different projects, have a look at [Per-Directory Loca
 Set `compilation-scroll-output`. For example, `(setq compilation-scroll-output t)`
 will turn on auto scrolling.
 
+### Multiple Compilation Buffer Support
```

Comment:
I'm not sure this is the right language to communicate this feature at the README-level, though I'm struggling to find a higher-level way to convey this feature. Will happily accept alternative suggestions.

Reply:
Personally, I'm okay with this wording. If nobody suggests a better wording in the next day of two, we'll merge as-is.